src/libraries/Directory.Build.targets

```diff
@@ -79,29 +82,34 @@
 
     <!-- Setup the shared framework directory for testing -->
     <BinPlaceTargetFrameworks Condition="'$(BinPlaceTestSharedFramework)' == 'true'" Include="$(NetCoreAppCurrent)-$(TargetOS)">
+      <NativePath>$(NETCoreAppTestSharedFrameworkPath)</NativePath>
       <RuntimePath>$(NETCoreAppTestSharedFrameworkPath)</RuntimePath>
     </BinPlaceTargetFrameworks>
     <BinPlaceTargetFrameworks Condition="'$(BinPlaceNETFXRuntime)' == 'true'" Include="$(NetFrameworkCurrent)-Windows_NT">
+      <NativePath>$(TestHostRootPath)</NativePath>
       <RuntimePath>$(TestHostRootPath)</RuntimePath>
     </BinPlaceTargetFrameworks>
 
-    <!-- Setup the runtime pack structure for mobile platforms (for testing and installer) -->
-    <!-- Corresponding assemblies are added to lib -->
-    <BinPlaceTargetFrameworks Condition="'$(BinPlaceTestRuntimePack)' == 'true'" Include="$(NetCoreAppCurrent)-$(TargetOS)">
-      <RuntimePath>$(RuntimePackLibDir)</RuntimePath>
+    <!-- Microsoft.NetCore.App.Ref targeting pack binplacing. -->
+    <BinPlaceTargetFrameworks Include="$(NetCoreAppCurrent)">
+      <RefPath>$(MicrosoftNetCoreAppRefPackRefDir)</RefPath>
     </BinPlaceTargetFrameworks>
-    <!-- Corresponding libMono and system.native libraries are added to native -->
-    <BinPlaceTargetFrameworks Condition="'$(BinPlaceTestRuntimePack)' == 'true'" Include="$(NetCoreAppCurrent)-$(TargetOS)">
-      <RuntimePath>$(RuntimePackNativeDir)</RuntimePath>
-      <ItemName>NativeBinPlaceItem</ItemName>
+
+    <!--
+      Setup the runtime pack structure for mobile platforms (for testing and installer)
+      Corresponding assemblies are added to lib
+      Corresponding libMono and system.native libraries are added to native
+    -->
+    <BinPlaceTargetFrameworks Include="$(NetCoreAppCurrent)-$(TargetOS)">
+      <NativePath>$(MicrosoftNetCoreAppRuntimePackNativeDir)</NativePath>
+      <RuntimePath>$(MicrosoftNetCoreAppRuntimePackRidLibTfmDir)</RuntimePath>
     </BinPlaceTargetFrameworks>
     <!-- Corresponding libMono and system.native libraries are added to testhost shared framework -->
-    <BinPlaceTargetFrameworks Condition="'$(BinPlaceTestRuntimePack)' == 'true' and '$(TargetsMobile)' != 'true'" Include="$(NetCoreAppCurrent)-$(TargetOS)">
+    <BinPlaceTargetFrameworks Condition="'$(RuntimeFlavor)' == 'Mono' and '$(TargetsMobile)' != 'true'" Include="$(NetCoreAppCurrent)-$(TargetOS)">
```

Comment:
I think @joperezr will need this for the linker tests, so maybe just enable it for once?

Reply:
Actually, since you're setting `NativePath` just above, we probably just don't need this property anymore?

Reply:
This is only about the testhost which doesn't affect the runtime pack.

Reply:
ah that's true. Thanks.